tests/cases/fourslash/codeFixPromiseToAsyncCatch.ts

```diff
@@ -0,0 +1,28 @@
+/// <reference path='fourslash.ts' />
+
+// @target: es6
+
+////function [|f|]():Promise<void> {
+////    return fetch('http://yahoo.com').then(result => { console.log(result); }).catch(err => { console.log(err); });
+////}
+
+verify.getSuggestionDiagnostics([{
+    message: "This may be converted to use async and await.",
+    code: 80006,
+}]);
+
+
+verify.codeFix({
+    description: "Convert to use async and await",
+    index: 0,
+    newFileContent:
+`async function f():Promise<void> {
+    try {
+        let result = await fetch('http://yahoo.com');
+        console.log(result);
+    }
+    catch (err) {

```

Comment:
Predominant JS style is catch on the same line as the closing brace. Style is called "One True Brace Style" or 1TBS.